### PR TITLE
bump sdp package to get RTCIceCandidate.address

### PR DESCRIPTION
**Pull request by fippo**
see https://github.com/w3c/webrtc-pc/issues/1913

### CODE REVIEW

**Comment by fippo** [comment]
probably minor.

File: test/e2e/rtcicecandidate.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ describe('RTCIceCandidate', () => {
           expect(hasProperty).to.equal(true);
           done();
         } else {
-          hasProperty = e.candidate.hasOwnProperty('port');
```
Comment (dagingaa) [inline test/e2e/rtcicecandidate.js:25]: Shouldn’t this have been ip?

File: package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "rtcpeerconnection-shim": "^1.2.10",
-    "sdp": "^2.7.0"
```
Comment (dagingaa) [inline package.json]: Intentional to bump by 2?

**Review by dagingaa — APPROVED** [review]

File: test/e2e/rtcicecandidate.js (hunk 1 above)
Comment (fippo) [reply test/e2e/rtcicecandidate.js:25]: no, it used to be port but checking for address is killing two with one stone

**Pull request merged**